Airports/EGKK/Positions.txt

```diff
@@ -6,5 +6,7 @@ EGKK-S_APP:Gatwick Director:126.820:KKS:S:EGKK~S:APP:-:-:3750:3761:N051.08.53.00
 EGKK_F_APP:Gatwick Director:118.950:KKF:K:EGKK:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
 EGKK_TWR:Gatwick Tower:124.220:KKT:K:EGKK:TWR:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
 EGKK_GND:Gatwick Ground:121.800:KKG:K:EGKK:GND:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK_P_GND:Gatwick Planner:134.220:KKP:K:EGKK:GND:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK_P_GND:Gatwick Planner:134.220:KKP:K:EGKK:DEL:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
```

Comment:
Should this be EGKK_P_DEL?

Reply:
Nope.
It's a GMP split but logs on as _GND to stop pilots calling them first by mistake.